hooks/output-filename.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+module.exports = {
+    'generate:after': generator => {
+        if(generator.templateParams.outFilename !== 'asyncapi.md') {
+            fs.rename(`${generator.targetDir}/asyncapi.md`,
```

Comment:
please use `fs.renameSync` so you do not have to pass this empty callback `() => {}`

Reply:
done in 6a647fb